markdown/solace-helm-node-samples/Helm-Node-Samples.md

```diff
@@ -75,23 +81,25 @@ helm install solace-node-sample . -f values.yaml
 
 ## Step 3: Run the Node.js Sample Applications
 
```

Comment:
Similar comment here, can you please add a Duration section `Duration: 0:02:00`. export and commit the changes after 👍

Reply:
Added a duration here as well.